specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2019-01-01-preview/cpimTenant.json

```diff
@@ -0,0 +1,917 @@
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "createTenantProperties": {
+              "x-ms-client-flatten": true,
+              "$ref": "#/definitions/CreateTenantProperties"
+            }
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/B2CResourceSKU"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource Tags"
+        }
+      }
+    },
+    "CreateTenantProperties": {
+      "type": "object",
+      "description": "These properties are used to create the Azure AD B2C tenant. These properties are not part of the Azure resource.",
+      "properties": {
+        "displayName": {
+          "description": "The display name of the B2C tenant.",
+          "type": "string"
+        },
+        "countryCode": {
+          "$ref": "#/definitions/CountryCode"
+        }
+      }
+    },
+    "CountryCode": {
+      "type": "string",
+      "description": "Country code of Azure tenant (e.g. 'US'). Refer to [aka.ms/B2CDataResidency](https://aka.ms/B2CDataResidency) to see valid country codes and corresponding data residency locations. If you do not see a country code in an valid data residency location, choose one from the list."
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      },
+      "description": "An error response for a resource management request."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
+      "x-ms-parameter-location": "method"
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the resource group.",
+      "x-ms-parameter-location": "method"
+    },
+    "ResourceNameParameter": {
+      "name": "resourceName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The initial domain name of the B2C tenant.",
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request.",
+      "x-ms-parameter-location": "method"
```

Comment:
This is more peculiar than the subscription id parameter, usually in a swagger file, all of the APIs defined in it are all using the api-version implied in this file which does not need an SDK user to specify explicitly (in this case it is `2019-01-01-preview`). It would bring confusion to the customers if they have to pass an api-version by themselves. Is this by design or a mistake?